### ISSUE TITLE
Some performance issues

### ISSUE THREAD

**Issue by giladdiv** [yhenon/keras-frcnn#211]
My results are not as good as I excpected so I whould like to hear some typical numbers so I could evaluate my model.
What learning rate are you using? whats the munber of ROI? Whats your mean number of roi over lapping the bbox and whats the accuracy for the classifier for the rpn. How much time does it take you for one epoch of 1000 iterations (I am using  1080Ti and its about 600 sec, is it slow?) and for how many epochs  do you train your model
I think that just copying the summary after a good epoch will do
Thanks

**Comment by taewookim** [comment]
600 secs on 1080 TI? WTH... i'm using Google GPU instance w/ nvidia k80 and 1 epoch of 1000 iterations is taking 1800'ish second. Time to get a real GPU i guess

**Comment by giladdiv** [comment]
@taewookim  with 32 rois? and it doesn't over-fit with 2000 images? after how many epochs does it converge?
Thanks!

**Comment by giladdiv** [comment]
@taewookim  sounds like overfit

**Comment by taewookim** [comment]
@giladdiv so what do you suggest? would rather not tinker with the architecture ... use keras-retinanet?

**Comment by giladdiv** [comment]
@taewookim  I wish I know how to help you I have the same problem. If you find something let me know
